src/base/bitset.cc

```diff
@@ -19,21 +20,23 @@ using namespace std;
 // on all platforms. Note that the positions are numbered 1 through 64, with
 // a return value of 0 indicating that there are no set bits.
 //
+#ifndef _WINDOWS //implemented in winutils.cpp
```

Comment:
This is inefficient. Please use https://msdn.microsoft.com/en-us/library/wfd9z0bb.aspx instead.